apiary.apib

```diff
@@ -25,7 +25,6 @@ API for requesting endorsements reports. Upon completion, the report will be del
                     + `endorsed_group` - Basic endorsed group report
                     + `endorsed_group_analytics` - Detailed analytics for endorsed group
                     + `transcripts` - Transcripts report
-                    + `quarterly` - Quarterly summary report
```

Comment:
Removing as we'll provide another endpoint for quarterly reporting as proposed in the tech design.